cmd/add/account/account.go

```diff
@@ -0,0 +1,80 @@
+package account
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+
+	"github.com/loginradius/lr-cli/cmdutil"
+
+	"github.com/spf13/cobra"
+)
+
+var fileName string
+
+type account struct {
+	NameString     string `json:nameString`
+	EmailString    string `json:emailString`
+	PasswordString string `json:passwordString`
+}
+
+func NewaccountCmd() *cobra.Command {
+	opts := &account{}
+
+	cmd := &cobra.Command{
+		Use:     "account",
+		Short:   "add account",
+		Long:    `This commmand adds account`,
+		Example: heredoc.Doc(`$ lr add account --name <name> --email <email> --password <password>`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.EmailString == "" {
+				return &cmdutil.FlagError{Err: errors.New("`email` is required argument")}
+			}
+			if opts.NameString == "" {
+				return &cmdutil.FlagError{Err: errors.New("`name` is required argument")}
+			}
+			if opts.PasswordString == "" {
+				return &cmdutil.FlagError{Err: errors.New("`password` is required argument")}
+			}
+
+			return add(*opts)
+
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringVarP(&opts.EmailString, "email", "e", "", "emailID")
+	fl.StringVarP(&opts.NameString, "name", "n", "", "name")
+	fl.StringVarP(&opts.PasswordString, "password string", "p", "", "password")
+
+	return cmd
+}
+
+func add(Account account) error {
+	res1, err := cmdutil.GetAPICreds()
+	var namestring = Account.NameString
+	var emailstring = Account.EmailString
+	var passwordstring = Account.PasswordString
+	url := "https://api.loginradius.com/identity/v2/manage/account?apikey=" + res1.Key + "&apisecret=" + res1.Secret
+	payload := strings.NewReader("{\n  \"FirstName\":\"" + namestring + "\",\n  \"Email\":[\n    {\n      \"Type\": \"Primary\",\n      \"Value\": \"" + emailstring + "\"\n    }\n  ],\n  \"Password\": \"" + passwordstring + "\"}")
```

Comment:
We already have the `account` struct, can't we use the same and covert it to JSON string?